policyengine/impact/population/charts.py

```diff
@@ -15,6 +15,40 @@
 from policyengine.utils.general import PolicyEngineResultsConfig
 
 
+def individual_decile_chart(
+    df: pd.DataFrame,
+    metric: str,
+) -> dict:
+    """Chart of average or relative net effect of a reform by income decile.
+
+    :param df: DataFrame with columns for Decile, Relative change, and Average change.
```

Comment:
No change needed here but I think we have some inconsistent docstring formats throughout the repo (mostly caused by me). FWIW I prefer the Google format as it's a bit more readable.